client/src/components/DataFiles/DataFilesModals/DataFilesCompressModal.js

```diff
@@ -0,0 +1,189 @@
+import React, { useMemo } from 'react';
+import { useSelector, useDispatch, shallowEqual } from 'react-redux';
+import {
+  Button,
+  Modal,
+  ModalHeader,
+  ModalBody,
+  ModalFooter,
+  Form,
+  Input,
+  FormGroup
+} from 'reactstrap';
+import { LoadingSpinner, FormField, Icon, InlineMessage } from '_common';
+import { useHistory, useLocation } from 'react-router-dom';
+import { isString } from 'lodash';
+import { Formik } from 'formik';
+import PropTypes from 'prop-types';
+import * as yup from 'yup';
+import './DataFilesCompressModal.module.scss';
+
+const makeDate = () => {
+  const date = new Date();
+  const d = new Date(date);
+  let month = `${d.getMonth() + 1}`;
+  let day = `${d.getDate()}`;
+  const year = d.getFullYear();
+
+  if (month.length < 2) month = `0${month}`;
+  if (day.length < 2) day = `0${day}`;
+
+  return [year, month, day].join('-');
+};
+
+const DataFilesCompressForm = ({ singleFile, formRef }) => {
+  const dateTimeStr = makeDate();
+  const initialValues = {
+    filenameDisplay: `${singleFile ? dateTimeStr : ''}`,
```

Comment:
This confuses me. I thought that these extensions required different commands to perform. And any other extension would also require its own command to create.

_I am only reading it because (A) I had a UI suggestion to show `.tar.gz`, not just `tar.gz` in the dropdown and (B) I am writing a PR for a branch that combines the extension selector and filename input into a single input group ([visual](https://user-images.githubusercontent.com/62723358/104661792-ed35c180-568e-11eb-8aad-24b5422f60d9.png))._

Reply:
They do require different commands - the parameters are passed to the app to decide what command to call. But I think we need to include the extension in the filename we send as well for this version of [zippy](https://bitbucket.org/taccaci/portal-agave-templates/src/master/cep/applications/zippy/wrapper.sh).

I like that visual 👍

Reply:
Ah, I mislead myself. _(**Note to Self**: the file extension comment is about data passing (after user completes modal) and the string input is about the file name input in the modal.)_